x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -0,0 +1,233 @@
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                defaultToEmptyTag(ip)
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.DOMAIN,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) => {
+      const domainAttr = `${topNFlowType}.domain`;
+      const ipAttr = `${topNFlowType}.ip`;
+      const domains: string[] = get(domainAttr, node);
+      const ip: string | null = get(ipAttr, node);
+
+      if (Array.isArray(domains) && domains.length > 0) {
+        const domain = domains[0];
+        const id = escapeDataProviderId(
+          `networkTopNFlow-table-${ip}-${topNFlowType}-${topNFlowDirection}-domain-${domain}`
+        );
+        return (
+          <DraggableWrapper
+            key={id}
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id,
+              name: domain,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: domainAttr,
+                value: domain,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                <>
+                  {domain}
+                  {domains.length > 1 && (
+                    <EuiToolTip
+                      content={
+                        <>
+                          {domains.slice(1, 6).map(domainName => (
+                            <span key={`${id}-${domainName}`}>
+                              {defaultToEmptyTag(domainName)}
+                              <br />
+                            </span>
+                          ))}
+                          {domains.slice(1).length > 5 && (
+                            <b>
+                              <br />
+                              more ...
+                            </b>
+                          )}
+                        </>
+                      }
+                    >
+                      <MoreDomains type="eye" />
+                    </EuiToolTip>
+                  )}
+                </>
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.DIRECTION,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.direction', node)
+        ? get('network.direction', node).length > 0
```

Comment:
Optional, I think you can save a ternary by using lodash `isEmpty()` as it will ensure each of these use cases.